cmd/thanos/sidecar.go

```diff
@@ -77,7 +78,7 @@ func registerSidecar(m map[string]setupFunc, app *kingpin.Application, name stri
 			objStoreConfig,
 			peer,
 			rl,
-			name,
```

Comment:
Same here - let's just put constant in place everywhere in sidecar - no point to dynamically eval this variable